packages/@react-stately/tabs/src/useTabListState.ts

```diff
@@ -69,11 +69,13 @@ function findDefaultSelectedKey<T>(collection: Collection<T> | null, disabledKey
   if (collection) {
     selectedKey = collection.getFirstKey();
     // loop over tabs until we find one that isn't disabled and select that
-    while (disabledKeys.has(selectedKey) && selectedKey !== collection.getLastKey()) {
+    // @ts-ignore
```

Comment:
I couldn't figure out the typescript error with `collection.getItem(selectedKey).props` which says `Property 'props' does not exist on type 'T'.ts` Wondering if someone else might have any ideas on how to fix this